fw/http_parser.c

```diff
@@ -5912,25 +5915,22 @@ Req_Method_1CharStep: __attribute__((cold))
 		__FSM_MOVE_nofixup_n(Req_MUSpace, 0);
 	}
 
-	__FSM_STATE(Req_UriRareForms, cold) {
+	__FSM_STATE(Req_UriAsteriskForm, cold) {
```

Comment:
Why did you rename it to `Req_UriAsteriskForm`? This state may lead to Asterisk and to Absolute forms. `Req_UriRareForms` is suitable name.